pkg/security/apiserver/securitycontextconstraints/matcher.go

```diff
@@ -58,8 +60,18 @@ func (d *defaultSCCMatcher) FindApplicableSCCs(namespace string, users ...user.I
 			}
 		}
 	}
-	sort.Sort(ByPriority(matchedConstraints))
-	return matchedConstraints, nil
+
+	internalMatchedConstraints := []*securityapi.SecurityContextConstraints{}
```

Comment:
Duplicate with external versions and use that also in the test that I've earlier mentioned. It'll be cheaper to maintain that, than do those conversions.

Reply:
Where do we use the internal sorter, is it still needed? Maybe nail a single place where it's being used and move it there and have a general purpose external sorter.

Reply:
@soltysh 
> Duplicate with external versions and use that also in the test that I've earlier mentioned. It'll be cheaper to maintain that, than do those conversions.

I don't think any of the consumers of this method (admission and rest.go) require dealing with external security constraints.
The reason for this conversion is to keep the signature of the method the same, since switching to an external lister means that calling `cache.List()` now returns external SecurityContextConstraints

Reply:
> Where do we use the internal sorter, is it still needed? Maybe nail a single place where it's being used and move it there and have a general purpose external sorter.

This makes sense. I believe that sorter is only used in this file and in tests. I'll go ahead and convert to use externals